src/Facebook/FacebookBatchRequest.php

```diff
@@ -216,12 +229,12 @@ public function validateBatchRequestCount()
      * Converts a Request entity into an array that is batch-friendly.
      *
      * @param FacebookRequest $request       The request entity to convert.
-     * @param string|null     $requestName   The name of the request.
-     * @param string|null     $attachedFiles Names of files associated with the request.
+     * @param array           $options       The name and the attached files of the request plus
+     *                                       the batch options e.g. 'omit_response_on_success'
      *
      * @return array
      */
-    public function requestEntityToBatchArray(FacebookRequest $request, $requestName = null, $attachedFiles = null)
+    public function requestEntityToBatchArray(FacebookRequest $request, array $options = [])
```

Comment:
Here is the BC break, because `requestEntityToBatchArray` is public, changing its signature is a BC break. :)